### DISCUSSION TITLE
How -to-remove-glossary-and-acronym-sections?

### DISCUSSION THREAD

**Discussion by vishu-byte** [joaomlourenco/novathesis#377] [Q&A] [answered]
I have tried almost everything, but still couldn't figure out how to get rid of glossary and acronym. I dont require these and therefore I need to remove them. I would be grateful if anyone can help.

Thanks

**Comment by joaomlourenco** [comment, marked as answer]
In file "Config/4_files.tex", comment the "\ntaddfile{…}" you want to skip.
```latex
\ntaddfile{glossaries}[glossary]{glossary}
\ntaddfile{glossaries}[acronyms]{acronyms}
\ntaddfile{glossaries}[symbols]{symbols}
\ntaddfile{glossaries}[symbols]{chemical}
```

**Reply by vishu-byte** [reply]
Yeah. I have already tried. I have commented these commands in "Config/4_files.tex", but still I could not get rid of these sections.

**Reply by vishu-byte** [reply]
All right. I got it. Thank you so much.
One more thing, how to skip copyright section that appears just after the title page.

**Comment by r-ribeiros** [comment]
hello, using latest version I am unable to remove symbols chapter, it is empty but still shows up as a chapter. Commenting "\ntaddfile{…}" does not work, what can I do? Thank you.

**Reply by joaomlourenco** [reply]
I cannot reproduce this behaviour.
Please tell exactly which version are you using. Thanks.

**Reply by r-ribeiros** [reply]
[Version 6.10.18] in overleaf

**Reply by joaomlourenco** [reply]
I still cannot reproduce this behaviour.
Please try the following:

- download the latest version
- upload it to overleaf as a new project
- comment the 4 ntaddfile commands in `Config/4_files.tex`
```latex
% \ntaddfile{glossaries}[glossary]{glossary}
% \ntaddfile{glossaries}[acronyms]{acronyms}
% \ntaddfile{glossaries}[symbols]{symbols}
% \ntaddfile{glossaries}[symbols]{chemical}
```
- Add the following to your `Config/5_packages.tex`
```latex
\renewcommand{\gls}[1]{\textbf{#1}}
\renewcommand{\glsfmtshort}[1]{\gls{#1}}
\renewcommand{\Gls}[1]{\gls{#1}}
```

And compile it.  Did you get the undesired page?

**Reply by r-ribeiros** [reply]
somehow it started working on its own, despite not changing anything else in the code; I just opened it now and symbols chapter is gone!
